src/material-examples/badge-overview/badge-overview-example.html

```diff
@@ -13,5 +13,9 @@
 <p>
   Icon with a badge
   <mat-icon matBadge="15" matBadgeColor="warn">home</mat-icon>
+  <!-- Include text description of the icon's meaning for screen-readers -->
+  <span class="cdk-visually-hidden">
+  Example with a home icon with overlaid badge showing the number 15
```

Comment:
Small nit: indent the content of the `span` by +2

Reply:
Yes sure. Do we have any code formatting configurations specific to Intelij Idea IDE.